akka-docs/src/main/paradox/typed/durable-state/persistence.md

```diff
@@ -152,6 +152,7 @@ and can be one of:
 * `persist` will persist the latest state. If it's a new persistence id, the record will be inserted. In case of an existing
 persistence id, the record will be updated only if the revision number of the incoming record is 1 more than the already
 existing record. Otherwise `persist` will fail.
+* `delete` will delete the state by setting it to the empty state and the revision number will not be incremented.
```

Comment:
ah, I didn't notice that before. Maybe it would be good to actually increment the revision number so that it is clear that the value changed?

Reply:
That makes sense to increment version. There is a unique version number for each change in payload from calling persist so incrementing version when calling delete would ensure that the version number for the deleted payload is not shared with the previous persist and payload.